### PR TITLE
Improve eHerkenning/DigiD settings

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Part of https://github.com/open-formulieren/open-forms/issues/1265

Pull request for the CLI arguments is here: https://bitbucket.org/maykinmedia/django-digid-eherkenning/pull-requests/20

### CODE REVIEW

**Review by sergei-maertens — APPROVED** [review]
I know too little about this to make a proper judgement, but code-wise this looks okay.

**Comment by joeribekker** [comment]
FYI: Before merging, I want to do a review :)

**Comment by sergei-maertens** [comment]
I intended to leave this to you indeed :)

File: src/openforms/conf/base.py
Comment (joeribekker) [inline src/openforms/conf/base.py:897]: Can we add an IF-statement to add the eIDAS part below dynamically? For example:

```python
if EIDAS_SERVICE_INDEX:
  EHERKENNING["services"].append(....)
```

Or does this cause other issues?

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: src/openforms/conf/base.py
Comment (SilviaAmAm) [reply src/openforms/conf/base.py:897]: I think it should be fine

**Review by joeribekker — APPROVED** [review]

**Pull request merged**
